src/mawpy/constants.py

```diff
@@ -1,13 +1,13 @@
 USER_ID = 'user_ID'
 STAY = 'stay'
 STAY_DUR = 'stay_dur'
-STAY_LAT = '             stay_lat' #TODO: currently sample csv shared (this is one of the examples with additional spaces
+STAY_LAT = 'stay_lat'
```

Comment:
Do we need a new/updated test file with this change?

Reply:
The last input csv file that Anurag shared is compatible with this.